src/data/int/basic.lean

```diff
@@ -1069,6 +1072,20 @@ units.ext_iff.1 $ nat.units_eq_one ⟨nat_abs u, nat_abs ↑u⁻¹,
 theorem units_eq_one_or (u : units ℤ) : u = 1 ∨ u = -1 :=
 by simpa only [units.ext_iff, units_nat_abs] using nat_abs_eq u
 
+lemma _root_.is_unit.eq_one_or {a : ℤ} : is_unit a → a = 1 ∨ a = -1
+| ⟨x, hx⟩ := hx ▸ (units_eq_one_or _).imp (congr_arg coe) (congr_arg coe)
+
+lemma is_unit_iff {a : ℤ} : is_unit a ↔ a = 1 ∨ a = -1 :=
+begin
+  refine ⟨λ h, is_unit.eq_one_or h, λ h, _⟩,
+  rcases h with rfl | rfl,
+  { exact is_unit_one },
+  { exact is_unit_of_mul_eq_one _ (- 1 : ℤ) (neg_one_mul _) }
+end
+
+theorem is_unit_int {n : ℤ} : is_unit n ↔ n.nat_abs = 1 :=
```

Comment:
I think a longer name might be better here:
```suggestion
lemma is_unit_iff_nat_abs_eq {n : ℤ} : is_unit n ↔ n.nat_abs = 1 :=
```

Reply:
Ok, I will change this, but with a push: the lemma is used twice in the rest of mathlib (or so it seems) and I will change all occurrences at once.